scripts/update-ts-configs.js

```diff
@@ -99,23 +100,23 @@ function writeRootTsConfigJson(pkgRoot, projectRoot, lernaProjects) {
   const tsconfig = readJSON(tsconfigPath);
   const references = Array.from(lernaProjects.values())
     .filter(it => it.isTsProject)
-    .map(it => path.relative(pkgRoot, path.join(projectRoot, it.dir))).sort();
+    .map(it => toPosix(path.relative(pkgRoot, path.join(projectRoot, it.dir)))).sort();
```

Comment:
Duplicated with line 105:

```suggestion
    .map(it => path.relative(pkgRoot, path.join(projectRoot, it.dir))).sort();
```